src/fastq/fastqread.jl

```diff
@@ -0,0 +1,64 @@
+# FASTQ Read
+# ==========
+#
+# Design discussions in https://github.com/BioJulia/FASTX.jl/pull/35
+
+mutable struct FASTQRead
+    identifier::String
+    description::String
+    sequence::BioSequences.LongDNASeq
+    quality::Vector{UInt8} # in raw PHRED scores, not offset
+end
```

Comment:
I think this should be immutable - in the unlikely even someone would like to rename a FASTQ read, they can just make another read.
Also, it would be nice if this was parameterized by an `BioSequences.Alphabet`, such that you could also have e.g. RNA reads, so something like
```
mutable struct FASTQRead{A <: Alphabet}
    identifier::String
    description::String
    sequence::BioSequences.LongSequence{A}
    quality::Vector{UInt8} # in raw PHRED scores, not offset
end
```

Reply:
Fixed in 3ce3e48